phpunit/class-block-templates-test.php

```diff
@@ -87,14 +87,6 @@ function test_inject_theme_attribute_in_content() {
 		);
 		$this->assertEquals( $expected, $template_content );
 
-		// Does not inject theme when there is an existing theme attribute.
-		$content_with_existing_theme_attribute = '<!-- wp:template-part {"slug":"header","theme":"fake-theme","align":"full", "tagName":"header","className":"site-header"} /-->';
```

Comment:
Instead of deleting this would it make sense to update it and test the case where `theme` attribute is not provided?